src/svelte5.svelte.js

```diff
@@ -0,0 +1,49 @@
+/** eslint-global: $state */
+import { mount, unmount } from 'svelte'
+
+import { SvelteTestingLibrary } from './pure.js'
+
+class Svelte5TestingLibrary extends SvelteTestingLibrary {
+  svelteComponentOptions = [
+    'target',
+    'props',
+    'events',
+    'context',
+    'intro',
+    'recover',
+  ]
```

Comment:
Hm I don't think this list is up to date with [the latest Svelte 5](https://github.com/sveltejs/svelte/blob/d408d20cddfb16899cf9607fcf0577838785c13f/packages/svelte/src/internal/client/render.js#L95-L102); probably worth a double-check outside of this PR

It looks like it's the list of options for `hydrate` rather than `mount`

- Missing `anchor`
- `recover` doesn't seem to be a valid option for `mount`